tdd-todo-list.CSharp.Main/Extension.cs

```diff
@@ -1,12 +1,100 @@
-﻿using System;
+﻿using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+using System;
 using System.Collections.Generic;
+using System.Diagnostics.Metrics;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace tdd_todo_list.CSharp.Main
 {
+    public struct Tasks
+    {
+        public string name;
+        public bool completed;
+        public int ID;
+        public int date;
+        public int time;
+    }
+
     public class TodoListExtension
     {
+        private List<Tasks> tasks = new List<Tasks>();
+
+        public void AddTask(string taskName, bool completed, int ID, int date, int time)
```

Comment:
The caller should only need to supply the task name. The rest should be internally generated